pkgs/applications/misc/kratos/default.nix

```diff
@@ -0,0 +1,42 @@
+{ fetchFromGitHub, buildGoModule, lib, stdenv }:
+
+buildGoModule rec {
+  pname = "kratos";
+  version = "v0.6.0-alpha.1.pre.2";
+
+  src = fetchFromGitHub {
+    owner = "ory";
+    repo = "kratos";
+    rev = "fa7fa701051c819ac74c919bc52721c57f662fb8";
+    sha256 = "1jyhb5cs532msqfgnhn6bnnqw6d94g3pjgirklbrysd3z4f32f8g";
+  };
+
+  vendorSha256 = "1n5g6mrr5c8z6y3ncpkap971n6m5qimpfqkq8vjgl8jkih4m282x";
+  buildPhase = ''
```

Comment:
You should

runHook preBuild

here, and 

runHook postBuild

at the end or you'll get confusing build failures.

Reply:
Fixed, thanks